python/ray/tune/suggest/bohb.py

```diff
@@ -15,7 +15,7 @@
 from ray.tune.suggest.suggestion import UNRESOLVED_SEARCH_SPACE, \
     UNDEFINED_METRIC_MODE, UNDEFINED_SEARCH_SPACE
 from ray.tune.suggest.variant_generator import parse_spec_vars
-from ray.tune.utils.util import flatten_dict, unflatten_dict
+from ray.tune.utils.util import flatten_dict, unflatten_dict, unflatten_list_dict
```

Comment:
I think handling `list`s should be done directly in `unflatten_dict` rather than creating a new function

what do you think @richardliaw?

Reply:
I think creating a new function is fine, actually!